cranelift/codegen/meta/src/isa/riscv64gc.rs

```diff
@@ -0,0 +1,25 @@
+use crate::cdsl::isa::TargetIsa;
+use crate::cdsl::settings::{SettingGroup, SettingGroupBuilder};
+
+use crate::shared::Definitions as SharedDefinitions;
+
+fn define_settings(_shared: &SettingGroup) -> SettingGroup {
+    let mut setting = SettingGroupBuilder::new("riscv64gc");
+
+    setting.add_bool("has_extension_m", "has extension M?", "", false);
+    setting.add_bool("has_extension_a", "has extension A?", "", false);
+    setting.add_bool("has_extension_f", "has extension F?", "", false);
+    setting.add_bool("has_extension_d", "has extension D?", "", false);
+
+    setting.add_bool("has_extension_v", "has extension V?", "", false);
+
+    setting.add_bool("has_extension_b", "has extension B?", "", false);
+
+    setting.add_bool("has_extension_zbkb", "has extension zbkb?", "", false);
+    setting.build()
```

Comment:
Maybe add presets for riscv64imac and riscv64gc? The x86_64 backend uses code like the following: https://github.com/bytecodealliance/wasmtime/blob/fd3ce1a6cb47b60440d5d35b0670e9139ccd8eb5/cranelift/codegen/meta/src/isa/x86.rs#L185-L189